.github/workflows/clients-dotnet.yml

```diff
@@ -26,9 +26,12 @@ jobs:
       - name: Install zig
         run: ./scripts/install_zig.sh
 
+      - name: Dotnet version
+        run: dotnet --version
+
       - name: Build dotnet
         working-directory: src/clients/dotnet
-        run: dotnet build --configuration Release
+        run: dotnet build --verbosity normal --configuration Release
```

Comment:
They give the verbosity level such a straightforward name `normal`, but at the same time, it's not the default option! OMG 🥴 !!

Nice to have this output if something fails.